packages/types/src/json.ts

```diff
@@ -273,6 +273,12 @@ export interface ClerkAPIErrorJSON {
     param_name?: string;
     session_id?: string;
     email_addresses?: string[];
+    zxcvbn?: {
+      suggestions: {
+        code: string;
+        message: string;
+      }[];
```

Comment:
❓ Do you think we should extract this in a separate type?

Reply:
@anagstef I believe being a bit more explicit here is no harm, so we can keep it as is.